libraries/wallet/wallet.cpp

```diff
@@ -4480,6 +4613,38 @@ string wallet_api::read_memo(const memo_data& memo)
    return my->read_memo(memo);
 }
 
+signed_message wallet_api::sign_message(string signer, string message)
+{
+   FC_ASSERT(!is_locked());
+   return my->sign_message(signer, message);
+}
+
+bool wallet_api::verify_message( string message, string account, int block, int time, compact_signature sig )
+{
+   return my->verify_message( message, account, block, time, sig );
+}
+
+/** Verify a message signed with sign_message
+ *
+ * @param message the signed_message structure containing message, meta data and signature
+ * @return true if signature matches
+ */
```

Comment:
Duplicating docs may lead to inconsistent docs.

Reply:
Removed.